flutter/lib/src/event_processor/screenshot_event_processor.dart

```diff
@@ -75,83 +75,34 @@ class ScreenshotEventProcessor implements EventProcessor {
       return event;
     }
 
-    final bytes = await _createScreenshot();
-    if (bytes != null) {
-      hint.screenshot = SentryAttachment.fromScreenshotData(bytes);
-    }
-    return event;
-  }
+    // ignore: deprecated_member_use
+    var recorder = ScreenshotRecorder(
+        ScreenshotRecorderConfig(quality: _options.screenshotQuality),
+        _options);
 
-  Future<Uint8List?> _createScreenshot() async {
-    try {
-      final renderObject =
-          sentryScreenshotWidgetGlobalKey.currentContext?.findRenderObject();
-      if (renderObject is RenderRepaintBoundary) {
-        // ignore: deprecated_member_use
-        final pixelRatio = window.devicePixelRatio;
-        var imageResult = _getImage(renderObject, pixelRatio);
-        Image image;
-        if (imageResult is Future<Image>) {
-          image = await imageResult;
-        } else {
-          image = imageResult;
-        }
-        // At the time of writing there's no other image format available which
-        // Sentry understands.
+    Uint8List? _screenshotCache;
```

Comment:
"cache" is a little misleading here. We don't cache it for future use

Reply:
yeah, you are right. thanks for the hint.